cmd/cli/handler_migrate.go

```diff
@@ -141,7 +141,7 @@ ROTATED_SYSTEM_SECRET=%s hydra serve ...
 
 If you wish that OAuth 2.0 Access and Refresh Tokens issued with the old secret are revoked, simply omit environment variable
 ROTATED_SYSTEM_SECRET. This will NOT affect OpenID Connect ID Tokens!
-`, newSecret)
+`, oldSecret)
```

Comment:
Line 138 `variable ROTATED_SYSTEM_SECRET to the new secret:` should also be `variable ROTATED_SYSTEM_SECRET to the **OLD** secret:`

Reply:
ping @prateek1192 :)

Reply:
Thanks @aeneasr :)